tests/unit/SchemaTest.php

```diff
@@ -75,7 +75,7 @@ public function getExpectedColumns()
         $columns['bool_col2']['precision'] = null;
         $columns['bool_col2']['scale'] = null;
         $columns['bool_col2']['defaultValue'] = true;
-        $columns['ts_default']['defaultValue'] = new Expression('now()');
+        $columns['ts_default']['defaultValue'] = 'CURRENT_TIMESTAMP';
```

Comment:
Why was this change needed? (As always, changing a test raises a red flag for me)

Reply:
Because ts_default value can not be "now()" in the way it is defined.

https://github.com/yiisoft/yii-core/blob/d3e3e142f52f231500749fe4c203056c882c595f/config/tests/postgres.sql#L141

Reply:
Is it really used? I think it is just not used.
Cause 'CURRENT_TIMESTAMP' will be used as string (with quoting) - `SELECT 'CURRENT_TIMESTAMP'` which is only a string in Postgres NOT a timestamp. 
`new Expression('now()')` will be used as expression (without quoting) this way `SELECT now()`.

So. It should be deleted if it is not used.
If it is used please show me how.

Reply:
It used here:

https://github.com/yiisoft/db/blob/e93aa758802befc85550e934d71c058f6dca0f7a/tests/unit/SchemaTest.php#L522